src/gstools/covmodel/base.py

```diff
@@ -530,14 +542,14 @@ def isometrize(self, pos):
         """Make a position tuple ready for isotropic operations."""
         pos = np.asarray(pos, dtype=np.double).reshape((self.field_dim, -1))
         if self.latlon:
-            return latlon2pos(pos)
+            return latlon2pos(pos, time=self.time)
```

Comment:
This behaves strange. I think we should add a `radius` attribute to CovModel to prevent the `rescale` hack to get a meaningful len_scale. At the moment the len_scale is given on the unit-sphere and that doesn't really work with the time-dimension and the associated len_scale.